images/macos/software-report/SoftwareReport.Common.psm1

```diff
@@ -632,6 +632,14 @@ function Build-GraalVMTable {
     }
 }
 
+function Get-PriorCodeQLBundleVersion {
```

Comment:
Do we really need these 2 functions? Given that we are using the same toolcache directory, I think we can simply iterate over versions with `For-Each` loop and then make something like `CodeQL versions: x.x.x, y.y.y`

Reply:
Thanks, I've made this change. Please take a look!